FWCore/ParameterSet/python/Config.py

```diff
@@ -1431,6 +1442,45 @@ def validate(self):
         #    raise RuntimeError("No input source was found for this process")
         pass
 
+    def handleProcessAccelerators(self, parameterSet):
+        allAccelerators = set([label for acc in self.__dict__['_Process__accelerators'].values() for label in acc.labels()])
```

Comment:
a trivial comment, this should also work without the temporary `list`:
```suggestion
        allAccelerators = set(label for acc in self.__dict__['_Process__accelerators'].values() for label in acc.labels())
```